rust/src/shards/gui/widgets/image.rs

```diff
@@ -159,6 +162,11 @@ impl Shard for Image {
 }
 
 impl Image {
+  fn get_scale(scale_var: &ParamVar, ui: &Ui) -> Result<egui::Vec2, &'static str> {
```

Comment:
Decided to draw images at the original size, so if an image is 512x512 it'll be drawn at that size regardless of the UI scale.
Maybe we need a flag for this though, where by default it does automatically scale but can be disabled.

Reply:
There was scaling params in the wrong imgui implmentation, why we don't follow that? please...

Reply:
Ah sorry with `UI Scale` I meant the scale the UI is drawn at relative to how many pixels it covers, e.g. 2x on mac displays, 1.5x if you set display scaling to 150% in windows settings.

The scale parameter is still applied, just that scale of 1.0 now corresponds to 1:1 image to screen pixels

Reply:
Undid that scaling change btw, so it's back to what it was as it makes more sense to just scale images with the UI by default. Only needed unscaled images for the render target widget which now does that by itself